deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -30,6 +30,9 @@ toolbox:
   tolerations: []
   # -- Toolbox affinity
   affinity: {}
+  # -- Toolbox container security context
+  containerSecurityContext:
+    runAsNonRoot: false
```

Comment:
Why default the security context to run as root? On the non-helm deployment, the security context on the toolbox is [here](https://github.com/rook/rook/blob/master/deploy/examples/toolbox.yaml#L90-L95), which is the same as the rook operator.

Reply:
I figured I'd keep the same settings the helm chart had before, but if the non-helm deployment has been tested with those settings I will update them to be the same.

(note: runAsNonRoot: false means k8s will not check the user, it does not enforce running as root)